02-spatial-data.Rmd

```diff
@@ -695,11 +696,38 @@ The simplest use-case for `**sfheaders**` is demonstrated in the code chunks bel
 - a matrix converted to `sfg_LINESTRING`
 - a data.frame converted to `sfg_POLYGON`
 
-```{r sfheaders-sfg_point} 
-v = c(1,1)
-sfheaders::sfg_point(obj = v)
+We will start by creating the simplest possible `sfg` object starting with a single coordinate pair, assigned to a vector names `v`:
+
+```{r}
+v = c(1, 1)
+v_sfg_sfh = sfheaders::sfg_point(obj = v)
+```
+
+```{r sfheaders-sfg_point, eval=FALSE} 
+v_sfg_sfh # printing without sf loaded
+#>      [,1] [,2]
+#> [1,]    1    1
+#> attr(,"class")
+#> [1] "XY"    "POINT" "sfg" 
 ```
 
+The example above shows the print methods that come with the **sfheaders** package, when **sf** is not loaded, demonstrating the underlying structures of the objects.
```

Comment:
sorry to be pedantic, and it's a minor technicality, but `sfheaders` does not expose a `print` method, so you're seeing the default R print method here.

Reply:
Good to be specific and yes it shows the 'raw' object. Hows the following...

Reply:
```
The example above shows how `sfg` objects are printed when **sf** is not loaded, demonstrating the underlying structures of the objects. 
```

Reply:
yeah that's good.